src/dagmc.cpp

```diff
@@ -279,39 +232,32 @@ void load_dagmc_geometry()
   }
 
   if (!graveyard) {
-    warning("No graveyard volume found in the DagMC model."
+    warning("No graveyard volume found in the DagMC model. "
```

Comment:
Do you need a graveyard even in a universe based model? Do OpenMC cells not bound this?

Reply:
The only case where you now need one is if the DAGMC model is the root universe of the Geometry. I'll add some detection for that so this warning doesn't appear when it shouldn't. Thanks!

Reply:
Reorganized the code a bit so we can perform this check in 302f12b5ffba88c78ce2cb39722e76ade9672735. Hopefully this is what you had in mind!